tooling/bundler/src/bundle/settings.rs

```diff
@@ -904,6 +906,12 @@ impl Settings {
     &self.bundle_settings.file_associations
   }
 
+  /// Return the list of deep link protocols to be registered for
+  /// this bundle.
+  pub fn deep_link_protocols(&self) -> &Option<Vec<DeepLinkProtocol>> {
+    &self.bundle_settings.deep_link_protocols
```

Comment:
Optional reference is better semantically than a reference to an option

```suggestion
  pub fn deep_link_protocols(&self) -> Option<&Vec<DeepLinkProtocol>> {
    self.bundle_settings.deep_link_protocols.as_ref()
```

Reply:
i copied this from the file associations. should i change that too?

Reply:
yeah